src/dev/getting_started.md

```diff
@@ -80,6 +75,11 @@ rustup override set yk-stage1
 Now `cargo` will run our compiler (also setting all of the various flags
 required) for this directory instead of the default Rust compiler.
 
+To select hardware tracing, we need to set an environment variable:
```

Comment:
s/we/you/ (to fit in with the rest of the file).

Reply:
Fixed in 3163fea.